assets/js/blocks/active-filters/edit.js

```diff
@@ -82,7 +82,7 @@ const Edit = ( { attributes, setAttributes } ) => {
 				onChange={ ( value ) => setAttributes( { heading: value } ) }
 			/>
 			<Disabled>
-				<Block attributes={ attributes } isPreview />
+				<Block attributes={ attributes } isEditor />
```

Comment:
Actually I thought the recommended way was to omit the `={ true }`, but it's true it might be confused with the ES6 object shorthand, good point.

Updated it in efd657c7d7145149ba52cea1e04819fb8be67e0a.